services/coolifyService.js

```diff
@@ -349,7 +377,7 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_rounded = name.allReplace(coolifyDicts.rounded);
 		let cool_name_square = name.allReplace(coolifyDicts.square);
 		let cool_name_emoji = name.allReplace(coolifyDicts.emoji);
-		let cool_name_round_alphanum = cool_name_rounded.slice(0, 1) + cool_name_alphanum.slice(1, name.length - 1)
+		let cool_name_round_alphanum = cool_name_rounded.slice(0, 1) + cool_name_alphanum.slice(1, name.length - 1)7he qu!(k brown fox jumped over 7he |@zy dog
```

Comment:
I guess you messed up 😄

Reply:
![screenshot from 2018-12-20 20-37-49](https://user-images.githubusercontent.com/36412264/50292461-2b079200-0497-11e9-8af3-5f40b8dcf40d.png)
Project successfully running after https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/e4beaaaf32b440e14af330599df6f27d7ccfb1e3